Advance_1/index.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+    /* Class to call the API. */
+    class API {
+        public $url;
+        function __construct($url) {
```

Comment:
Add comments to this function as well as specify the `access modifier`. Mention the type of parameter.